.github/workflows/pytest.yml

```diff
@@ -88,6 +89,12 @@ jobs:
           pip list
         shell: bash
 
+      - name: Install Neuropod backend
+        run: |
+          sudo mkdir -p "$NEUROPOD_BASE_DIR"
+          curl -L https://github.com/uber/neuropod/releases/download/v0.3.0-rc6/libneuropod-cpu-linux-v0.3.0-rc6-torchscript-1.10.2-backend.tar.gz | sudo tar -xz -C "$NEUROPOD_BASE_DIR"
```

Comment:
Suggest including ENVARS in this curl request  for future upgrades:
* `NEUROPOD_VERISON:  "0.3.0-rc6"`
* `TORCHSCRIPT_VERISON: "1.10.2"`

```
https://github.com/uber/neuropod/releases/download/v${{ env.NEUROPOD_VERISON }}/libneuropod-cpu-linux-v${{ env.NEUROPOD_VERISON }}-torchscript-${{ env.TORCHSCRIPT_VERISON }}-backend.tar.gz
```